src/cardCollectionConcept.java

```diff
@@ -0,0 +1,38 @@
+import components.map.Map1L;
+
+public class cardCollectionConcept {
+    private String card;private Map<
+    String date,
+    int value>cardValues;
+
+    public cardCollectionConcept(String cardName) {
+        this.card = cardName;
+        this.cardValues = new Map1L<>();
+    }
+
+    public String getCard() {
+        return this.card;
+    }
+
+    public void setDateValue(String date, int value) {
+        this.cardValues.add(date, value);
+    }
+
+    public int getValueAtDate(String date) {
+        this.cardValues.value(date);
+    }
+
+    public static void main(String[] args) {
+        cardCollectionConcept card1 = new cardCollectionConcept("Pikachu");
+
+        String d1 = "10/12/2021", d2 = "12/18/2024";
```

Comment:
This is great. Take a look at the DateTime object in Java, which might serve you better.